.github/workflows/wheel_linux_aarch64_cuda.yml

```diff
@@ -92,12 +86,34 @@ jobs:
 
           CIBW_BUILD_VERBOSITY: 3
 
-        run: python3 -m cibuildwheel --output-dir wheelhouse
+        run: python -m cibuildwheel --output-dir wheelhouse
+
+      - name: Determine Python version
+        id: pyvs
+        shell: bash
+        run: |
+          echo "version=$(echo  ${{ matrix.cibw_build }} | tr -cd '[:digit:].' | sed 's/./&./1')" >> $GITHUB_OUTPUT
+
+      - uses: actions/setup-python@v5
+        name: Install Python
+        with:
+          python-version: ${{ steps.pyvs.outputs.version }}
+
+      - name: Test wheels
```

Comment:
Do we need to test with min and max Python versions?

Reply:
Let's check both for now and revisit it later given the time we have.